collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -977,6 +986,12 @@ def query_factory(name, version=NO_VERSION):
             ['replslot_wal_keep', 'wal keeped', 'absolute'],
             ['replslot_files', 'pg_replslot files', 'absolute']
         ]
+    },
+    'stat_replications': {
+        'options': [None, 'Stat Replications', 'Replications db', 'stat replication', 'postgres.stat_replication', 'line'],
```

Comment:
What is 'Stat Replications'? (i checked [MONITORING-PG-STAT-REPLICATION-VIEW](https://www.postgresql.org/docs/13/monitoring-stats.html#MONITORING-PG-STAT-REPLICATION-VIEW)) Could you explain this chart/metrics?

Reply:
My point is - it supposed to have a specific name (e.g.: current_wal_sender_state),  stat replications doesn't say much.

Reply:
changed to db_replications